src/sentry/static/sentry/app/views/settings/components/dataScrubbing/modals/form/sourceField.tsx

```diff
@@ -235,7 +235,9 @@ class SourceField extends React.Component<Props, State> {
     for (const index in fieldValues) {
       const fieldValue = fieldValues[index];
       if (Array.isArray(fieldValue)) {
-        newValue.push(`${fieldValue[0].value}${fieldValue[1].value}`);
+        if (fieldValue[0]?.value || fieldValue[1]?.value) {
+          newValue.push(`${fieldValue[0]?.value}${fieldValue[1]?.value}`);
```

Comment:
Is it possible that this will end up as a string with 'undefined' inside of it?

Reply:
good catch.. I've updated the logic 😉